gitclone.adoc

```diff
@@ -17,6 +17,6 @@ git clone https://github.com/openliberty/guide-{projectid}.git
 cd guide-{projectid}
 ----
 
-The `start` directory contains the starting project that you can use to begin.
+The `start` directory contains the starting project that you will build upon.
```

Comment:
What about, "...you **can** build upon." to stick with the present tense? However, I'm OK with how it currently is, though, because the readers will be building as they proceed with the guide.

Reply:
yeah, thats the premise that I went with, the user will be doing something so I used future tense

Reply:
OK, we can leave it as it is. :)

Reply:
ok, thanks @sishida :D

Reply:
Thanks :) Merging